### PR TITLE
Fix/vps reverse ip

### PR DESCRIPTION
**Pull request by marie-j**
### Description of the Change

* Use /vps/serviceName/ips only for legacy VPS

MANAGER-3009 

### Related

- [x] https://github.com/ovh-ux/ovh-api-services/pull/192

### CODE REVIEW

File: client/app/ip/ip/reverse/update/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "ip_table_manage_reverse": "Modifier le reverse",
+  "ip_table_manage_reverse_question": "Insérez un reverse valide pour cette IP, et cliquez sur \"Valider\".",
```
Comment (antleblanc) [inline client/app/ip/ip/reverse/update/translations/Messages_fr_FR.json]:
```suggestion
  "ip_table_manage_reverse_question": "Insérez un reverse valide pour cette IP et cliquez sur \"Valider\".",
```

Does it make sense to keep the comma separator?

Comment (antleblanc) [reply client/app/ip/ip/reverse/update/translations/Messages_fr_FR.json]: I noticed that you've only moved theses existing translations. But I think the suggestion could be taken in account.

Comment (marie-j) [reply client/app/ip/ip/reverse/update/translations/Messages_fr_FR.json]: I agree it can be removed :)

**Comment by antleblanc** [comment]
@marie-j :tada: `ovh-api-services@7.1.0` has been released.

**Pull request merged**